### PR TITLE
Api2000 network

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Describe what this change achieves]

### Issues Resolved
[]

### Check List
- [ ] New functionality includes testing.
  - [x]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/fcoe_network.tf [hunk 1]
```diff
@@ -6,26 +6,27 @@ provider "oneview" {
...
-resource "oneview_fcoe_network" "FCoENetwork" {
-	name = "TestFCoENetwork_Terraform"
-	type = "fcoe-networkV4"
-  vlanId = 202
-  initial_scope_uris = ["${data.oneview_scope.scope_obj.uri}", "${data.oneview_scope.scope_obj1.uri}"]
-}
+# resource "oneview_fcoe_network" "FCoENetwork" {
+# 	name = "TestFCoENetwork_Terraform"
+# 	type = "fcoe-networkV4"
+#   vlanId = 202
```
Comment (AsisBagga) [inline examples/fcoe_network.tf:20]: please align the arguments properly

File: examples/fcoe_network.tf [hunk 2]
```diff
@@ -6,26 +6,27 @@ provider "oneview" {
...
+#   initial_scope_uris = ["${data.oneview_scope.scope_obj.uri}", "${data.oneview_scope.scope_obj1.uri}"]
```
Comment (AsisBagga) [inline examples/fcoe_network.tf:21]: same as above

File: examples/fcoe_network.tf [hunk 3]
```diff
@@ -6,26 +6,27 @@ provider "oneview" {
...
+# }
 
 # Updates the resource created above
 # To update uncomment the below and add the attributes to be updated
 /*resource "oneview_fcoe_network" "FCoENetwork" {
 	name = "TestFCoENetwork_Terraform_Renamed"
 	type = "fcoe-networkV4"
-  vlanId = 202
+   vlanId = 202
```
Comment (AsisBagga) [inline examples/fcoe_network.tf:29]: same as above

File: examples/fcoe_network.tf (hunk 2 above)
Comment (nabhajit-ray) [reply examples/fcoe_network.tf]: Done

File: examples/fcoe_network.tf (hunk 1 above)
Comment (nabhajit-ray) [reply examples/fcoe_network.tf]: Done

**Review by chebroluharika — APPROVED** [review]

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**